mne/cov.py

```diff
@@ -903,7 +947,7 @@ def _gaussian_loglik_scorer(est, X, y=None):
     n_samples, n_features = X.shape
     log_like = np.zeros(n_samples)
     log_like = -.5 * (X * (np.dot(X, precision))).sum(axis=1)
-    log_like -= .5 * (n_features * log(2. * np.pi) - _logdet(precision))
+    log_like -= .5 * (n_features * np.log(2. * np.pi) - _logdet(precision))
```

Comment:
why was this necessary? I prefer to use math module for operations on native types such as float